src/radical/pilot/utils/db_utils.py

```diff
@@ -208,7 +110,7 @@ def get_session_events(db, sid, cache=None, cachedir=None):
 
     """
 
-    docs = get_session_docs(db, sid, cache, cachedir)
+    docs = get_session_docs(sid, cachedir)
```

Comment:
if `db` is not provided then need to set the name for `cachedir` explicitly: `get_session_docs(session, cachedir=cachedir)`

Reply:
fixed.